docs/reference/copy-guide.md

```diff
@@ -0,0 +1,275 @@
+# Gutencopy Guidelines
+
+## Longer Text
+Guidelines for writing multi-line/step instructions or narrative introductions/orientation to pages or features.
+
+This will obviously vary quite a lot depending on the context, but here are some general tips:
+
+#### ONE: Contractions are your friends!
+They’re more conversational, and a simple way to make text sound friendlier and less formal. (And they save a bit of space as well: a win-win.)
+
+#### TWO: Cut phrases that inflate your word count without actually adding meaning. 
+This happens frequently in two specific instances. First, when writing in the passive voice:
+
+> This block can be used to display single images.
+
+Any time you see phrases like “can be” or “is used”: halt. You’re writing in the passive voice. Try going active for a snappier (and shorter) sentence:
+
+> This block displays single images.
+
+Second, this happens when we hedge instead of making declarative statements:
+
+> The gallery block can help you display multiple images in an elegant layout.
+
+Does it or doesn’t it? We’re making this software: we’re allowed to be declarative about what it is and does:
+
+> The gallery block displays multiple images in an elegant layout.
+
+We also all do this a lot with the phrase “allows you to.” 
+
+> Preformatted text allows you to keep your tabs and line breaks.
+
+Features don’t allow anyone to do anything; they’re just tools that do specific things to achieve an end. Just say what they do:
+
+> Preformatted text preserves your tabs and line breaks.
+
+The more direct sentences are almost always clearer. Scan your copy for the words “can,” “be,” “might,” “allows you to,” and “helps” -- they’re the most common culprits, and looking for those words specifically is a way to locate phrasing you can tighten up. 
+
+#### THREE: Beware of “simple,” “easy,” and “just.” 
```

Comment:
The commas should go outside the quotes.

Reply:
I'm not sure what WordPress.org punctuation standards are, but WordPress.com standards (which is where I come from) are U.S. punctuation, where they go inside the quotes.

Reply:
Even in a list like that? Whoa… I mean whatever the style guide says… but that's extremely difficult for me to read.

I thought for quotes in prose (eg quoting someone) they went inside, but that looks like a list; I'd expect the commas after each item, not inside it.

I think this will confuse programmers 😉

🤷‍♂️

Reply:
The rules are a bit uncertain when it comes to instances where quotation marks are used for object titles rather than actual quotes. Here is a Quora page I found after a quick search:

https://www.quora.com/When-writing-a-list-in-which-the-items-are-in-quotations-should-the-comma-go-inside-the-quotes-or-outside

I am fairly certain my English textbooks would always put commas after the quotation marks in a list of words. Off the top of my head, that seems to be the standard that I see used in most places, but I could be misremembering. Whatever the case, have commas within the quotation marks in this instance looks more confusing to me.

Perhaps the issue could be avoided entirely by simply italicizing the words and avoiding the usage of quotation marks?

Reply:
Italics seem like a great solution :)